front/src/applications/operationalStudies/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -119,6 +129,7 @@ const handleTrainrunOperation = async ({
           },
         })
       ).unwrap();
+      upsertTrainSchedules([newTrainSchedule]);
```

Comment:
I believe this triggers a series of pathfinding/simulation requests down the line. I think this is undesirable because it will spam the backend with requests every time a small change is made in NGE.